staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/metrics/metrics.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	namespace = "apiserver"
+	subsystem = "encryptionconfig_controller"
+)
+
+var (
+	encryptionConfigHotReloadFailureTotal = metrics.NewCounter(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "encryption_config_hot_reload_failures_total",
```

Comment:
```suggestion
			Name:           "hot_reload_failures_total",
```

This metric would be `apiserver_encryptionconfig_controller_encryption_config_hot_reload_failures_total` which has `encryption_config` twice.